core/keyserver/keyserver.go

```diff
@@ -591,6 +596,7 @@ func (s *Server) BatchQueueUserUpdate(ctx context.Context, in *pb.BatchQueueUser
 	}
 	if wm != nil {
 		watermarkWritten.Set(float64(wm.Watermark), directory.DirectoryID, fmt.Sprintf("%v", wm.LogID))
+		sequencerQueueWritten.Add(float64(len(in.Updates)), directory.DirectoryID, fmt.Sprintf("%v", wm.LogID))
```

Comment:
Optional: it may be useful to include another counter to capture the number of updates this method receives regardless of whether they are successfully written. The ratio of these counters would tell you how large an impact any errors are causing. Suggest keyserver_queue_received.

Reply:
I'll do that in a followup. Thanks!